packages/core/src/messages/responses/response-base.ts

```diff
@@ -36,7 +36,7 @@ export interface IResponseSubscription {
 }
 
 export abstract class BaseResponseError extends ResponseBase {
-  public _innerException: SdkError;
+  protected readonly _innerException: SdkError;
```

Comment:
Did you mean for this change to be in the PR?

Reply:
yeah